.pre-commit-hooks.yaml

```diff
@@ -9,7 +9,7 @@
   entry: >
     (?x)(
       \bpath\(\s*["'][^"']+(?<!\/)["'],|
-      (url|re_path)\(\s*r?["'][^'"]+(?<!\/\$)(?<!\/)["'],
+      (url|re_path)\(\s*r?["'][^'"]+(?<!\/\$)(?<!\/)(?<!\.\*\)\$)["'],
```

Comment:
It's only for when the pattern is a regex one that ends in `.*`. The tests show examples. In these cases appending a slash can be incorrect because it would look for `//` at the end.

Reply:
https://regex101.com/r/qmPgXR/1

Reply:
Right right right I was missing a layer of regex. I see now 👍